{ 'cebit_weight_lift': { 'head': [],
                         'left_arm': [ { 'duration': 4.008,
                                         'name': u'cebit_weight_4',
                                         'positions': [ -2.2126,
                                                        -0.394,
                                                        -0.2014,
                                                        0.0237,
                                                        0.0391,
                                                        -0.1268,
                                                        -0.0457],
                                         'starttime': 0.0},
                                       { 'duration': 4.008,
                                         'name': u'cebit_weight_1',
                                         'positions': [ -0.8654,
                                                        0.0188,
                                                        -0.2016,
                                                        0.0235,
                                                        -1.5611,
                                                        0.0342,
                                                        -0.0525],
                                         'starttime': 4.008}],
                         'left_leg': [],
                         'right_arm': [ { 'duration': 4.008,
                                          'name': u'cebit_weight_4',
                                          'positions': [ 2.2126,
                                                         0.394,
                                                         0.2014,
                                                         -0.0237,
                                                         -0.0391,
                                                         0.1268,
                                                         0.0457],
                                          'starttime': 0.0},
                                        { 'duration': 4.008,
                                          'name': u'cebit_weight_1',
                                          'positions': [ 0.8654,
                                                         -0.0188,
                                                         0.2016,
                                                         -0.0235,
                                                         1.5611,
                                                         -0.0342,
                                                         0.0525],
                                          'starttime': 4.008}],
                         'right_leg': [],
                         'torso': []}}
